src/utils/string.ts

```diff
@@ -79,11 +79,12 @@ export function trimSuffix(str: string, suffix: string, limit?: number): string
  */
 export function trimUriPrefix(str: string, prefix: string): string {
   const shortPrefix = trimSuffix(prefix, "/");
-  if (str.startsWith(prefix)) {
+  const strLower = str.toLowerCase();
+  if (strLower.startsWith(prefix)) {
     // longPrefix is exactly at the beginning
     return str.slice(prefix.length);
   }
-  if (str.startsWith(shortPrefix)) {
+  if (strLower.startsWith(shortPrefix)) {
```

Comment:
We should `toLower` the `shortPrefix` before comparing in order to have truly case-insensitive comparison.

Reply:
or we can use regex with case insensitive flag instead of transforming the string

Reply:
Regex seems overkill in terms of complexity and slowness. At least, my instinct is that a regex solution would be slower but I haven't benchmarked it.